sycl/include/CL/sycl/backend.hpp

```diff
@@ -242,7 +245,8 @@ typename std::enable_if<
 template <backend Backend, typename T, int Dimensions = 1,
           typename AllocatorT = buffer_allocator>
 typename std::enable_if<detail::InteropFeatureSupportMap<Backend>::MakeBuffer ==
-                            true,
+                                true &&
+                            Backend != backend::ext_oneapi_level_zero,
```

Comment:
What is the motivation for excluding L0 here?

Reply:
The check for removing the L0 backend has been added to the [file](https://[github.com](https://github.com/intel/llvm/blob/8edb62e5c10e1ba22b355225a6f4ccd09bdf098d/sycl/include/CL/sycl/backend.hpp#L244)/intel/llvm/blob/8edb62e5c10e1ba22b355225a6f4ccd09bdf098d/sycl/include/CL/sycl/backend.hpp#L244) in a recent commit. This was carried to this build since I used the latest version for this PR. 

I didn't notice this change at the time I was committing this change, thanks for pointing it out. I'll revert it back to the original.